assets/js/modules/analytics/datastore/index.js

```diff
@@ -42,71 +42,15 @@ const baseModuleStore = Modules.createModuleStore( 'analytics', {
 	],
 } );
 
-export const INITIAL_STATE = Data.collectState(
-	baseModuleStore.INITIAL_STATE,
-	accounts.INITIAL_STATE,
-	properties.INITIAL_STATE,
-	profiles.INITIAL_STATE,
-	settings.INITIAL_STATE,
-	tags.INITIAL_STATE,
+const store = Data.combineStores(
+	baseModuleStore,
+	accounts,
+	properties,
+	profiles,
+	settings,
+	tags,
 );
 
-export const actions = Data.collectActions(
-	baseModuleStore.actions,
-	accounts.actions,
-	properties.actions,
-	profiles.actions,
-	settings.actions,
-	tags.actions,
-);
-
-export const controls = Data.collectControls(
-	baseModuleStore.controls,
-	accounts.controls,
-	properties.controls,
-	profiles.controls,
-	settings.controls,
-	tags.controls,
-);
-
-export const reducer = Data.addInitializeReducer(
-	INITIAL_STATE,
-	Data.collectReducers(
-		baseModuleStore.reducer,
-		accounts.reducer,
-		properties.reducer,
-		profiles.reducer,
-		settings.reducer,
-		tags.reducer,
-	)
-);
-
-export const resolvers = Data.collectResolvers(
-	baseModuleStore.resolvers,
-	accounts.resolvers,
-	properties.resolvers,
-	profiles.resolvers,
-	settings.resolvers,
-	tags.resolvers,
-);
-
-export const selectors = Data.collectSelectors(
```

Comment:
Please restore these exports similar to the others.

Reply:
:+1: